exchanges/lbank/lbank_wrapper.go

```diff
@@ -198,62 +197,46 @@ func (l *Lbank) UpdateTicker(ctx context.Context, p currency.Pair, a asset.Item)
 
 // UpdateOrderbook updates and returns the orderbook for a currency pair
 func (l *Lbank) UpdateOrderbook(ctx context.Context, p currency.Pair, assetType asset.Item) (*orderbook.Base, error) {
-	if p.IsEmpty() {
-		return nil, currency.ErrCurrencyPairEmpty
+	if !l.SupportsAsset(assetType) {
+		return nil, fmt.Errorf("%w: %v", asset.ErrNotSupported, assetType)
```

Comment:
🐒 This would work, right?
```suggestion
		return nil, fmt.Errorf("%w: %q", asset.ErrNotSupported, assetType)
```